src/tpm2_wrap.c

```diff
@@ -2059,6 +2060,82 @@ int wolfTPM2_ReadPublicKey(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* key,
 
 #ifndef WOLFTPM2_NO_WOLFCRYPT
 #ifndef NO_RSA
+int wolfTPM2_RsaPrivateKeyImportDer(WOLFTPM2_DEV* dev,
+    const WOLFTPM2_KEY* parentKey, WOLFTPM2_KEYBLOB* keyBlob, const byte* input,
+    word32 inSz, TPMI_ALG_RSA_SCHEME scheme, TPMI_ALG_HASH hashAlg)
+{
+    int rc = 0;
+    RsaKey key[1];
```

Comment:
You are missing the RSA init and free.

Reply:
I thought we didn't need that since we're not doing any operations with the key, we're basically just using the `mp_int` members of the key and those are all static

Reply:
It's more complicated, I don't need to init because decode will call init on the the `mp_int` members but it's it's too complicated to tell if the `mp_int` members alloc or not so I'll free just to be safe

Reply:
nevermind, the free call has problems if init wasn't called so I'll just do it